package/ios/RNSkia-iOS/RNSkDrawViewImpl.h

```diff
@@ -28,16 +28,16 @@ class RNSkDrawViewImpl : public RNSkia::RNSkDrawView {
   void setSize(int width, int height);
 
 protected:
-  int getWidth() override { return _width * _context->getPixelDensity(); };
-  int getHeight() override { return _height * _context->getPixelDensity(); };  
+  float getScaledWidth() override { return _width * _context->getPixelDensity(); };
```

Comment:
The methods are now called `getScaledWidth`/`getScaledHeight` in the base common class so it is consistent on all platforms since these methods now expects a return value containing the actual number of pixels. Maybe `getPixelWidth` / `getPixelHeight` would be better?